config/checker_profile_map.json

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
I think we need to update the user guide too: https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#checker-profiles
Checker profiles are no longer in the `config.json` file but in this new file.